deployment/deploy.sh

```diff
@@ -117,6 +125,12 @@ function  create_scheduler() {
     echo
 }
 
+function pause_scheduler() {
+    start_message "Pausing scheduler $1..."
+    gcloud scheduler jobs pause --location=$LOCATION $1
```

Comment:
Using the same location for the scheduler and bq_dataset will not work in some cases. For instance, datasets can be located in US, while schedulers ought to be region specific (e.g. us-central1)

Reply:
Hi Pedro,

I have addressed the comments, thank you!